src/scripts/parse-xls.py

```
@@ -0,0 +1,118 @@
+                        # pad blanks
+                        while last_col_idx + 1 < col_idx:
+                            row_data.append("")
+                            last_col_idx += 1
+                        last_col_idx = col_idx
+                        
+                    v_node = c.find('.//{*}v')
+                    if v_node is not None and v_node.text is not None:
+                        if c.attrib.get('t') == 's':
+                            try:
+                                val = shared_strings[int(v_node.text)]
+                            except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)